frontend/src/api/k8s/routes.ts

```diff
@@ -14,3 +14,14 @@ export const getRoute = (
       queryOptions: { name, ns: namespace },
     }),
   );
+
+export const getServiceMeshGwHost = async (namespace: string): Promise<string | null> => {
+  const queryOptions = {
+    name: namespace,
+  };
+  const project = await k8sGetResource<NamespaceKind>({ model: NamespaceModel, queryOptions });
```

Comment:
nit: In the frontend we tend to use Promises (.then()/.catch()) instead of async await, just as general rule.

Reply:
Also, if you want to get the project, don't use `NamespaceKind`, we already have `ProjectKind` in place